client-mr/src/test/java/org/apache/hadoop/mapreduce/RssMRUtilsTest.java

```diff
@@ -74,6 +75,22 @@ public void blockConvertTest() {
     assertEquals(taskAttemptId, newTaskAttemptId);
   }
 
+  @Test
```

Comment:
Could use `@ParameterizedTest` here.

Reply:
For example (remove the outer `for` loop):

```java
@ParameterizedTest
@ValueSource(ints = {0, 1, 233, 1023, 1024, 1234, (1 << Constants.PARTITION_ID_MAX_LENGTH) - 1})
public void partitionIdConvertBlockTest(int partitionId) {
```

Reply:
I think it's ok now. Let's merge this pr first.